hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/IndexListenerService.kt

```diff
@@ -126,14 +148,23 @@ class IndexListenerService(
       }
   }
 
-  private fun sync(prisonerNumber: String, eventType: String): Prisoner? =
+  /**
+   * Sync both the old green/blue and new red indices
+   */
+  private fun syncBoth(prisonerNumber: String, eventType: String): Prisoner? =
     nomisService.getOffender(prisonerNumber)?.run {
-      reindexPrisoner(ob = this, eventType)
+      reindexPrisonerNew(ob = this, eventType)
+      reindexPrisonerOld(ob = this, eventType)
     } ?: null.also { log.warn("Sync requested for prisoner {} not found", prisonerNumber) }
 
-  private fun sync(bookingId: Long, eventType: String): Prisoner? =
+  private fun syncOld(prisonerNumber: String, eventType: String): Prisoner? =
```

Comment:
might be clearer if this was `syncGreenBlue` and the other was `syncRed` for now

Reply:
Same applies to the `reindexPrisoner...` methods.

Reply:
yes good idea